examples/diagnose/plot_estimator_report.py

```diff
@@ -0,0 +1,338 @@
+"""
+============================================
+Get insights from any scikit-learn estimator
+============================================
+
+This example shows how the :class:`skore.EstimatorReport` class can be used to
+quickly get insights from any scikit-learn estimator.
+"""
+
+# %%
+#
+# TODO: we need to describe the aim of this classification problem.
```

Comment:
@GaelVaroquaux @jeromedockes Could you provide me a bit more details on what is the purpose of this dataset in `skrub`. It would be nice to give a short introduction to the ML problem we try to solve.